frontend/common/src/components/Link/ScrollToLink.tsx

```diff
@@ -18,6 +19,7 @@ const ScrollToLink = ({ to, children, ...rest }: ScrollToLinkProps) => {
   const handleClick = (e: React.MouseEvent<HTMLAnchorElement>) => {
     if (targetSection) {
       e.preventDefault();
+      HISTORY.push(`#${to}`);
```

Comment:
Should maybe use `navigate` from router.tsx instead of HISTORY.push directly?

Reply:
I did not realize that is what `navigate` was doing 😳 

Changed in: [use navigate in favour of history](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4216/commits/b5ed0942be7e329ffdffb2da691b9df0126ead30)